src/hooks/useTokenBalances.ts

```diff
@@ -90,6 +90,7 @@ export const useTokenBalances = (): UseTokenBalanceResult => {
 
   useEffect(() => {
     // can return NULL (if no address or network)
+    walletInfo.isConnected &&
```

Comment:
Doesn't networkId and userAdress imply isConnected?

Reply:
yes, but there's no userAddress here
Actually in case with MMask we can have userAddress after we _disconnect_ wallet , because there's no real disconnect for MMask